images/csi-driver-smb.yml

```
@@ -0,0 +1,29 @@
+content:
+  source:
+    dockerfile: Dockerfile.openshift
+    git:
+      branch:
+        target: release-{MAJOR}.{MINOR}
+      url: git@github.com:openshift-priv/csi-driver-smb.git
+      web: https://github.com/openshift/csi-driver-smb
+    ci_alignment:
+      streams_prs:
+        ci_build_root:
+          stream: rhel-9-golang-ci-build-root
+        auto_label:
+        - approved
+        - lgtm
+distgit:
+  branch: rhaos-{MAJOR}.{MINOR}-rhel-9
+  component: csi-driver-smb-container
+enabled_repos:
+- rhel-9-baseos-rpms
+- rhel-9-appstream-rpms
+for_payload: false
+from:
+  builder:
+  - stream: rhel-9-golang
+  member: openshift-enterprise-base-rhel9
+name: openshift/ose-csi-driver-smb-rhel9
```

```suggestion
name: openshift/ose-csi-driver-smb-rhel9
name_in_bundle: smb-csi-driver-container-rhel9
```
to match https://github.com/openshift/csi-operator/blob/master/config/samba/manifests/preview/image-references#L10